includes/debug-delivery-metrics.md

```diff
@@ -0,0 +1,3 @@
+## Delivery metrics
+
+The events delivered, not delivered, and delivery rate metrics for this destination are an estimate based on a sample of events sent to the Google Analytics 4 Measurement Protocol validation endpoint. In addition to sending all events to the ingestion endpoint, a sample of up to the first 100 events (and 100 user property changes, if "Send Users" is enabled) are sent to the validation endpoint every 10 minutes. The metrics shown in Amplitude are scaled from the sample of events, sent to the validation endpoint, to the total volume of events sent to the ingestion endpoint. If the delivery rate is below 100%, the counts of events delivered shown in Amplitude may not match exactly with the number of events seen in Google Analytics 4. Latency metrics are still computed exclusively from events sent to the ingestion endpoint.
```

Comment:
Should this maybe have a link?
>Google Analytics 4 Measurement Protocol validation endpoint

Reply:
Sure. I can add links for both the validation and ingestion endpoints.